app/navigation/Navigator.scala

```diff
@@ -16,29 +16,43 @@
 
 package navigation
 
-import javax.inject.{Inject, Singleton}
-import play.api.mvc.Call
+import config.Constants
+import config.Constants.MaxSchemeBankAccounts
+import config.Refined.OneToTen
 import controllers.routes
-import pages.{CheckReturnDatesPage, _}
+import eu.timepit.refined._
+import eu.timepit.refined.api.Refined
 import models._
+import pages._
+import play.api.mvc.Call
+
+import javax.inject.{Inject, Singleton}
 
 @Singleton
 class Navigator @Inject()() {
 
-
-
   private val normalRoutes: Page => UserAnswers => Call = {
     case StartPage(srn)             => _ => routes.SchemeDetailsController.onPageLoad(srn)
     case SchemeDetailsPage(srn)     => _ => routes.CheckReturnDatesController.onPageLoad(srn, NormalMode)
 
     case page@CheckReturnDatesPage(srn)  => ua =>
       if(ua.get(page).contains(true))
-        routes.SchemeBankAccountController.onPageLoad(srn, NormalMode)
+        routes.SchemeBankAccountController.onPageLoad(srn, refineMV[OneToTen](1), NormalMode)
```

Comment:
Should this alaways go to the first bank account? Maybe if there is data it should go to the list page. Maybe ask Josh or Michael

Reply:
Spoke to design team and they said it should just go to the list page